etc/releasing.md

```diff
@@ -214,22 +214,31 @@ git push --set-upstream origin releases/v1.2
 
 The new branch should be continuously tested on Evergreen. Update the "Display Name" and "Branch Name" of the [mongo-cxx-driver-latest-release Evergreen project](https://spruce.mongodb.com/project/mongo-cxx-driver-latest-release/settings/general) to refer to the new release branch.
 
-## Generate and Publish Documentation
+## Create Documentation Tickets
 
 Documentation generation must be run after the release tag has been made and
 pushed.
 
-- Create and checkout a new branch to contain documentation updates from master: `git checkout -b post-release-changes master`. This branch will be used to create a PR later.
+- Create and checkout a new branch to contain documentation updates from master:
+  `git checkout -b post-release-changes master`. This branch will be used to
+  create a PR later.
 - Edit `etc/apidocmenu.md` and add the released version in the `mongocxx` column
   following the established pattern. If this is a minor release (X.Y.0), revise
   the entire document as needed.
-- Edit `docs/content/_index.md` and `README.md` to match.
-- Edit the `Installing the MongoDB C driver` section of
-  `docs/content/mongocxx-v3/installation/advanced.md` to reflect libmongoc
-  requirements.
-- Edit `docs/content/mongocxx-v3/installation/linux.md`,
-  `docs/content/mongocxx-v3/installation/macos.md` and
-  `docs/content/mongocxx-v3/installation/windows.md`.
+- Edit the `README.md` to match.
+- Ensure [DOCSP JIRA
+  tickets](https://jira.mongodb.org/projects/DOCSP/issues/DOCSP-37927?filter=allopenissues)
+  are created for:
+
+  - the `Installing the MongoDB C driver` section of the [Advanced Installation
+   documentation](https://github.com/mongodb/docs-cpp/blob/master/source/installation/advanced.txt)
+  to reflect libmongoc requirements
+  - the installation pages for:
+
+    - [Linux](https://github.com/mongodb/docs-cpp/blob/master/source/installation/linux.txt)
+    - [Windows](https://github.com/mongodb/docs-cpp/blob/master/source/installation/windows.txt)
+    - [macOS](https://github.com/mongodb/docs-cpp/blob/master/source/installation/macos.txt)
+
    If the release was not a release candidate, update `Step 2` to reflect the
    new latest stable version to download.
```

Comment:
Suggest limiting this step to stable (not release candidate) releases. Suggest recommending creating DOCSP ticket to simplify the process. Note updating version in snooty.toml and home page table:

```suggestion
- If the release was not a release candidate, ensure a [DOCSP JIRA ticket](https://jira.mongodb.org/projects/DOCSP/issues/) is created to request updating:

  - the `Installing the MongoDB C driver` section of the [Advanced Installation documentation](https://github.com/mongodb/docs-cpp/blob/master/source/installation/advanced.txt) to reflect libmongoc requirements
  - the `Driver Status by Family and Version` section of the [home page](https://github.com/mongodb/docs-cpp/blob/master/source/installation/linux.txt)
  - the [C++ driver version](https://github.com/mongodb/docs-cpp/blob/e13d787967172220ae19e5d78df61e2071735f0f/snooty.toml#L20-L21).
```